javascript/javascript3/week2/Promise-exercises/ReWriting-exercise/rewritePromises.js

```diff
@@ -0,0 +1,26 @@
+function setTimeoutPromise(delay) {
+    return new Promise((resolve) => {
+            setTimeout(resolve, delay * 1000)
+        });
+    }
+
+setTimeoutPromise(4)
+    .then(() => {
+        console.log(`Called after 4 seconds.`);
+    });
+
+function getCurrentLocation() {
+        return new Promise((resolve,reject) => {
+            navigator.geolocation.getCurrentPosition(resolve, reject)
+        })
+        }    
```

Comment:
watch the syntax 👀 the indention is a little messy. Your IDE (code editor) should be able to help with that.

The code is good though 👍 


Very nice job keeping `getCurrentLocation` as simple as possible, by passing in `resolve` and `reject` as the `getCurrentPosition` success/error callbacks. Would have done the same 💯 it makes it very reusable.
```
function getCurrentLocation() {
  return new Promise((resolve, reject) => {
    navigator.geolocation.getCurrentPosition(resolve, reject)
  })
}
```